<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Periksa</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tanggal</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dtpTanggal">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Dokter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCariDokter">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cari Dokter</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtKodeDokter">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNamaDokter">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNamaPasien">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pasien</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtKodePasien">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCariPasien">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cari Pasien</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Keluhan</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txtKeluhan">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>481</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblObat">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>671</width>
     <height>192</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Kode Obat</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nama Obat</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Jumlah Obat</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Harga Obat</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Sub Total</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnTambahObat">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cari Obat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSimpan">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>650</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnKeluar">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>650</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Keluar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtKodeObat">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNamaObat">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>370</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtJml">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>370</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtHargaObat">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>370</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnInsertObat">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>360</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Tambah Obat</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>600</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Biaya</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtTotalBiaya">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>600</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
